niceday-api/api/openapi.yaml

```
@@ -193,3 +206,76 @@ components:
           description: Whether the tracker should be enabled
           example: true
 
+    TrackerReminder:
+      description: Reminder for a user tracker
+      required:
+      - userId
+      - recurringSchedule
+      type: object
+      properties:
+        userId:
+          type: string
+          description: ID of the user for which the reminder has to be set
+          example: "38527"
+        recurringSchedule:
+          $ref: '#/components/schemas/RecurringSchedule'
+
+    RecurringSchedule:
+      description: Recurring expressions object
+      required:
+      - title
+      - schedule_type
+      - recurring_expression
+      type: object
+      properties:
+        title:
+          type: string
+          description: Titile of the schedule
+          example: 'Smoking entry reminder'
+        schedule_type:
+          type: string
+          description: Type of schedule to set
+          example: 'tracker_smoking'
```

```suggestion
        tracker_id:
          type: string
          description: The id of the tracker to set the reminder for
          example: 'tracker_smoking'
```
I guess this is what you mean right?

---

But if you want to mimic their api, then add least update the description.